test/integration/features/proxy.feature

```diff
@@ -0,0 +1,87 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+
```

Comment:
So how do I run this test. I figure I have to set some environment variables, but I am not sure which ones exactly. I got asked for a password in the middle of the test . I thought it was the CDK registration and set MINISHIFT_USER and MINISHIFT_PASSWORD. But now I am wondering whether this was the password for the proxy.

Maybe the feature header should mention which env variables this test expects to be set.